lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -2757,6 +2799,98 @@ static void configureAIEVecCommonLegalizations(ConversionTarget &target,
     target.addIllegalOp<vector::TransferReadOp>();
   }
   target.addIllegalOp<vector::ExtractStridedSliceOp>();
+
+  // ****************************NEW****************************
+  target.addDynamicallyLegalOp<arith::ExtFOp>([](arith::ExtFOp extfOp) {
```

Comment:
So basically this is saying that extf ops are legal, unless that they are from v16 bfloat16  to v16 f32.

Reply:
I honestly don't know what the logic is here, I haven't changed the semantics of what we do with these ops, just where it happens and what assumptions they're making about order and other patterns.

It'd probably be a good idea, at some point, to take a good look at all these casts (and movements between accumulator/regular register files) and see what needs to be done. There's probably a sensible way to not do any of this, and simplify the conversion.